soh/soh/SaveManager.cpp

```diff
@@ -480,9 +479,9 @@ void SaveManager::SaveRandomizer(SaveContext* saveContext, int sectionID, bool f
             SaveManager::Instance->SaveData("hintedCheck", hint->GetHintedLocation());
             SaveManager::Instance->SaveData("hintType", hint->GetHintType());
             SaveManager::Instance->SaveStruct("hintedRegion", [&]() {
-                SaveManager::Instance->SaveData("english", hint->GetHintedRegionText().GetEnglish());
-                SaveManager::Instance->SaveData("french", hint->GetHintedRegionText().GetFrench());
-                SaveManager::Instance->SaveData("german", hint->GetHintedRegionText().GetEnglish());
+                SaveManager::Instance->SaveData("english", ::Hint(hint->GetHintedArea()).GetText().GetEnglish());
+                SaveManager::Instance->SaveData("french", ::Hint(hint->GetHintedArea()).GetText().GetFrench());
+                SaveManager::Instance->SaveData("german", ::Hint(hint->GetHintedArea()).GetText().GetEnglish());
```

Comment:
You can just store the RandomizerArea enum value here. As mentioned above, `LoadRandomizerVersion3` is new for develop-rando, so you can freely break compatibility here and in that function until it ends up in a release. You do still need to make sure the format in LoadRandomizerVersion2 and 1 gets parsed into the new objects, but for LoadRandomizerVersion3 and SaveRandomizer anything goes.